### PR TITLE
New commit to fix PEP8 formatting in issue 6 and to merge with new ch…

### PR DESCRIPTION
**Pull request by julianser**
…anges in Fuel. @vdumoulin

### CODE REVIEW

**Comment by dwf** [comment]
https://scrutinizer-ci.com/g/mila-udem/fuel/inspections/16765e0a-d74d-4630-9c77-ce182423c4ac

File: fuel/converters/cifar100.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    h5file.close()
+
+
+def fill_subparser(subparser):
+    """Sets up a subparser to convert the CIFAR100 dataset files.
+
+    Parameters
+    ----------
+    subparser : :class:`argparse.ArgumentParser`
+        Subparser handling the `cifar100` command.
```
Comment (vdumoulin) [inline fuel/converters/cifar100.py]: You're missing a blank line after this line, this is why Scrutinizer complains.

**Comment by vdumoulin** [comment]
Currently the tests fail because the CIFAR100 dataset isn't downloaded when Travis builds.

Add `cifar100` to [this line](https://github.com/mila-udem/fuel/blob/master/.travis.yml#L40), and we should be okay.

**Comment by julianser** [comment]
@vdumoulin I think there is a merge mistake. I am looking into it...

**Comment by vdumoulin** [comment]
I think it may just be a punctual failure. I restarted the tests, we'll see what happens.

**Comment by julianser** [comment]
Yes, it must be, because I just ran:

fuel-download cifar100
fuel-convert cifar100

They both worked fine...

**Comment by vdumoulin** [comment]
The tests now pass. Go figure...

The only thing left to do now is to make sure we maintain test coverage. There are two things to test: the download script and the conversion script.

This involves creating tests like the [cifar10 downloader test](https://github.com/mila-udem/fuel/blob/master/tests/test_downloaders.py#L100-L109) and the [cifar10 converter test](https://github.com/mila-udem/fuel/blob/master/tests/test_converters.py#L220-L279).

You can pretty much copy-paste those tests and adapt them for CIFAR100.

**Comment by vdumoulin** [comment]
Looking back at the converter test, it's more involved than adapting the CIFAR10 test. Let's leave the coverage as it is and make the converter test as part of another PR.

Once the tests pass, I'll merge.

**Pull request merged**